### PR TITLE
Fix some toString methods in presto-orc

### PR DESCRIPTION
**Pull request by sdruzkin**
## Description
Fix some incorrectly formatted toString methods.

## Test Plan
mvn test

## Contributor checklist

- [x] Please make sure your submission complies with our [development](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#development), [formatting](https://github.com/prestodb/presto/wiki/Presto-Development-Guidelines#formatting), [commit message](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), and [attribution guidelines](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#attribution).
- [x] PR description addresses the issue accurately and concisely.  If the change is non-trivial, a GitHub Issue is referenced.
- [x] Documented new properties (with its default value), SQL syntax, functions, or other functionality.
- [x] If release notes are required, they follow the [release notes guidelines](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines).
- [x] Adequate tests were added if applicable.
- [x] CI passed.

## Release Notes

```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Review by elharo — APPROVED** [review]

**Comment by sdruzkin** [comment]
@ajaygeorge can you please stamp this PR? Elliotte does not have committer status for presto-orc.

**Comment by elharo** [comment]
It's annoying that you can't merge this very simple and obviously correct and safe PR even though you are a committer on presto-orc. Really, presto is way too picky about who must stamp and approve what. Both Google and Meta have less stringent rules than this for their much larger and more diverse internal repositories, and it works just fine.

**Review by NikhilCollooru — APPROVED** [review]

**Pull request merged**
